docs/assets/style.css

```diff
@@ -4,6 +4,10 @@ body {
   font-family: 'Poppins', sans-serif;
 }
 
+a {
+  text-decoration: none;
+}
+
```

Comment:
Note: only removes underline from links - need to alter colours later.  Ok for now,.